waterlog/src/Components/JokeComponent.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { fetchNotifications } from '../actions/NumberOfNotifsActions';
+
+
+class Joke extends Component {
+  componentDidMount() {
+    this.timer = setInterval(()=>  this.props.fetchNotifications(), 5000);
+  }
+  componentWillUnmount() {
```

Comment:
Space between declarations

Reply:
Thanks this file was not used, I deleted it.